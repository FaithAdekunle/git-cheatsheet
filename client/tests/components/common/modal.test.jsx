import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Modal from '../../../src/app/components/common/modal';

describe('<ModalOverlay />', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<Modal children={{}} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
