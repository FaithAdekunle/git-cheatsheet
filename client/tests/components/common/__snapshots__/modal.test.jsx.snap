// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalOverlay /> should match snapshot 1`] = `
<Fragment>
  <div
    className="modal-overlay"
  />
  <div
    className="app-modal"
  >
    <div
      className="app-modal-content-wrapper"
    >
      <Component />
    </div>
  </div>
</Fragment>
`;
